modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,160 @@
+
+.. Enter a volume set name and a storage class name.
+
+.. Choose *All nodes* or *Select nodes* to apply filters accordingly.
++
+[NOTE]
+====
+Only worker nodes are available, regardless of whether you filter using *All nodes* or *Select nodes*.
+====
++
+.. Select the disk type, mode, size, and limit you want to apply to the local volume set, and click *Create*.
++
+A message displays after several minutes, indicating that the "Operator reconciled successfully."
+
+[start=3]
+. Alternatively, to provision local volumes for the discovered devices from the CLI:
+
+.. Create an object YAML file to define the local volume set, such as `local-volume-set.yaml`, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: local.storage.openshift.io/v1alpha1
+kind: LocalVolumeSet
+metadata:
+  name: example-autodetect
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+      - matchExpressions:
+          - key: kubernetes.io/hostname
+            operator: In
+            values:
+              - worker-0
+              - worker-1
+  storageClassName: example-storageclass <1>
+  volumeMode: Filesystem
+  fstype: ext4
+  maxDeviceCount: 10
+  deviceInclusionSpec:
+    deviceTypes: <2>
+      - disk
+      - part
+    deviceMechanicalProperty:
+      - NonRotational
+    minSize: 10G
+    maxSize: 100G
+    models:
+      - SAMSUNG
+      - Crucial_CT525MX3
+    vendors:
+      - ATA
+      - ST2000LM
+status:
+  conditions:
+  - lastTransitionTime: "2020-10-12T16:48:47Z"
+    message: 'DiskMaker: Available, LocalProvisioner: Available'
+    status: "True"
+    type: DaemonSetsAvailable
+  - lastTransitionTime: "2020-10-12T16:48:47Z"
+    message: Operator reconciled successfully.
+    status: "True"
+    type: Available
+  observedGeneration: 1
+  totalProvisionedDeviceCount: 0
+----
++
+<1> Determines the storage class that is created for persistent volumes that are provisioned from discovered devices.
++
+<2> When using the local volume set feature, the Local Storage Operator does not support the use of logical volume management (LVM) devices.
+
+.. Create the local volume set object:
++
+[source,terminal]
+----
+$ oc apply -f local-volume-set.yaml
+----
+
+.. Verify that the local persistent volumes were dynamically provisioned based on the storage class:
++
+[source,terminal]
+----
+$ oc -n get pv
+----
++
+.Example output
+[source,terminal]
+----
+NAME                CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM   STORAGECLASS           REASON   AGE
+local-pv-1cec77cf   100Gi      RWO            Delete           Available           example-storageclass            88m
+local-pv-2ef7cd2a   100Gi      RWO            Delete           Available           example-storageclass            82m
+local-pv-3fa1c73    100Gi      RWO            Delete           Available           example-storageclass            48m
+----
+
+[NOTE]
+====
+Results are deleted after they are removed from the node. Symlinks must be manually removed.
+====
```

Comment:
This does not look right. What result we are talking about? Can we remove this note and update the section about - "Deleting the Local Storage Operator resources" and update that section to also include `LocalVolumeSet` objects?

Reply:
Okay. I have removed this note here and added a note in the "Removing" section you suggested: 

> The following procedure outlines an example for removing a local volume. The same procedure can also be used to remove symlinks for a local volume set custom resource.